apps/provider.c

```diff
@@ -134,12 +136,16 @@ static void print_caps(META *meta, INFO *info)
         BIO_printf(bio_out, "%*s%s ", meta->indent, "", meta->label);
         print_method_names(bio_out, info);
         BIO_printf(bio_out, "\n");
+        print_param_types("settable keygen parameters",
+                          info->gen_settable_params, meta->subindent);
```

Comment:
Side note: gettable keygen params seems conspicuous by its absence.

Reply:
I did look at it while doing this. It does not seem to be a useful operation.
What you can set on the keygen is, and the gettable on the key after a gen is..